src/types/index.ts

```diff
@@ -9,15 +9,18 @@ export type AnyFunction = (...args: unknown[]) => unknown
 export type Mutation<T> = (original: T) => T
 export type Unpromise<T> = T extends Promise<infer U> ? U : T
 
+//This Enum here is temporary until SupportedChainId is exported
 export enum Network {
-  Mainnet = 1,
-  Ropsten = 3,
-  Rinkeby = 4,
-  Goerli = 5,
-  Kovan = 42,
-  xDAI = 100,
+  MAINNET = 1,
+  RINKEBY = 4,
+  GNOSIS_CHAIN = 100,
```

Comment:
Was it necessary to make it capital case?
No strong preferences, just that it caused a lot of noise in this PR.

If this is coming from the SDK, all good.

Reply:
Yup, We have defined that way in the SDK. Once `SupportedChainId` type is exported in the SDK with the new version bump, I'll update it to use the type instead